lib/js/jsre.nim

```diff
@@ -53,11 +53,26 @@ func contains*(pattern: cstring; self: RegExp): bool =
 
 
 runnableExamples:
-  let jsregex: RegExp = newRegExp(r"\s+", r"i")
-  jsregex.compile(r"\w+", r"i")
-  assert "nim javascript".contains jsregex
-  assert jsregex.exec(r"nim javascript") == @["nim".cstring]
-  assert jsregex.toCstring() == r"/\w+/i"
-  jsregex.compile(r"[0-9]", r"i")
-  assert "0123456789abcd".contains jsregex
-  assert $jsregex == "/[0-9]/i"
+  block:
+    let jsregex: RegExp = newRegExp(r"\s+", r"i")
+    jsregex.compile(r"\w+", r"i")
```

Comment:
IMO you should create new variables and separate these examples with empty lines for greater clarity.